web_app/tests/test_user.py

```diff
@@ -122,15 +120,15 @@ async def test_change_user_contract(
     ],
 )
 async def test_get_user_contract_address(
-    async_client, wallet_id: str, expected_contract_address: str
+    wallet_id: str, expected_contract_address: str
 ):
     """
-    Test get_user_contract_address endpoint,
-    where wallet_id = "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49" or ""
-    :param async_client: httpx.AsyncClient
+    Test get_user_contract_address endpoint
+    :param wallet_id: "" or "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49"
+    :param expected_contract_address: "" or "0x698b63df00be56ba39447c9b9ca576ffd0edba0526d98b3e8e4a902ffcf12f0"
     :return: None
     """
-    response = await async_client.get(
+    response = client.get(
         url="/api/get-user-contract-address",
         params={
             "wallet_id": wallet_id,
```

Comment:
The same thing here:
   assert response_json.get("contract_address") == `contract_address`